js/utils/providers/AZLyrics.js

```diff
@@ -8,8 +7,8 @@ import { parseDOM } from '../DOMUtils';
  */
 
 const COMMENT_REGEX = /<!--.*?-->((.|\n)*)/;
-const LYRIC_SELECTOR = 'div.col-xs-12.col-lg-8.text-center > div:nth-of-type(5)';
-const URL = metadata => `https://www.azlyrics.com/lyrics/${metadata.artist}/${metadata.title}.html`;
+const URL = metadata => `https://CelebratedRadiantCareware--lordraj.repl.co/lyrics?title=${metadata.title}&artist=${metadata.artist}`;
```

Comment:
Is this guaranteed not to change?

Reply:
Yep, the name is the name you choose for your project on repl.it, I guess I should have chosen a more meaningful name instead of letting it generate a random one.

Reply:
Perhaps we should rename URL to something more indicative of what the URL does. Happy to leave it tho

Reply:
Yea I guess I should have done that before merging will submit another PR at some point in the near future